src/libs/modals/modals/ModalTokenList/useModalTokenList.ts

```diff
@@ -75,6 +75,17 @@ export const useModalTokenList = ({ id, data }: Props) => {
     [tokens, excludedTokens, includedTokens]
   );
 
+  const duplicateSymbols = useMemo(() => {
+    return Object.values(
+      tokens.reduce((r: { [symbol: string]: string[] }, v) => {
+        (r[v.symbol] ??= []).push(v.symbol);
+        return r;
+      }, {})
+    )
+      .filter((g) => g.length > 1)
+      .flat();
```

Comment:
This is quite complex. Consider using a for loop : 
```ts
const isDuplicate = {};
const duplicates = new Set();
for (const symbol of Object.values(tokens)) {
  if (isDuplicate[symbol]) duplicates.add(symbol);
  isDuplicate[symbol] = true;
}
return Array.from(duplicates);
```

Reply:
That's right - this is much simpler. Done